### DISCUSSION TITLE
Changelog in About should point to the changelog of that release, not the latest

### DISCUSSION THREAD

**Discussion by hunleyd** [home-assistant/frontend#13152] [Other feature requests] [answered]
Loving the new About page, but the Changelog link takes you to the latest Changelog, not the Changelog for the release you're running. As a contrived example, I just updated to 2022.07.6 even though 2022.08.0 was just release (because I'm slow take upgrades). I want to see what changed _in the version I'm running_ hence going to the About page (where everything is related to what I'm running). As long as I'm running 2022.07.x. I should be taken to https://www.home-assistant.io/blog/2022/07/06/release-20227/

**Comment by frenck** [comment, marked as answer]
It cannot do that currently, as the link blog post links are not deterministic.

Instead, we could either link to the overview page listing all release notes blogs or change the name of the menu item to: "Latest changelog" to match what you'll actually get.

**Reply by hunleyd** [reply]
If you're running 2022.7.x then you know the blog is at /blog/2022/07/first_wednesday_of_month/release-20227 no? And 20228.x would be /blog/2022/08/first_wednesday_of_month/release-20228 right?

Oh, but if the release wasn't on the first Wed for some reason, this would fall over. 

Maybe a little mod_rewrite magic on the main site to redirect /release-20227 to the full URL and then About could just point to /release-20227 ?

**Reply by frenck** [reply]
There is no mod rewrite we can use. This has all been discussed many time before, we simply have no way to do this reliably
Same for linking directly to patch release versions.

**Reply by hunleyd** [reply]
Well that's unfortunate. Thanks for the replies @frenck .
